src/utils/FileRecordTools/FileReaders/SingleLineDelimTextFileReader.cpp

```diff
@@ -153,7 +153,7 @@ int SingleLineDelimTextFileReader::getVcfSVlen() {
 		//comma found in the number, that means there are two numbers
```

Comment:
Couldn't there be more than 2 numbers, in general? A typical SVLEN description in a VCF header from http://samtools.github.io/hts-specs/VCFv4.2.pdf allows for arbitrary number of lengths.